k-distribution/tests/regression-new/context-alias-2/test.k

```diff
@@ -0,0 +1,24 @@
+module TEST
+  imports INT
+  imports ID
+  imports MAP
+
+  syntax Exp ::= foo(Exp, Exp) [seqstrict(left; 1; right; 2)]
+               | Id "=" Int
+               | Id
+               | Int
+               | l(Exp) [symbol] | r(Exp) [symbol]
+  syntax KResult ::= Int
```

Comment:
Please write some more tests:

-   One which uses 3 context aliases (eg `left`, `right`, and `center`)
-   One which uses the same alias for multiple argument positions: (eg `left; 1, 3; right; 2`)